### PR TITLE
Control API documentation

### LINKED ISSUE #22: Control API doc please
As one who deploys ConductR, I want documentation for the Control Server so I can listen to bundles and members events in order to build management solutions for my deployment.

### PR DESCRIPTION
**Pull request by huntc**
Fixes #22

### CODE REVIEW

File: src/main/play-doc/developer/Control-API.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Typesafe ConductR %PLAY_VERSION%
+
+
+## The Control API
+
+ConductR's control protocol is RESTful and has the following functional scope:
+
+* load a bundle and optionally its configuration
```
Comment (viktorklang) [inline src/main/play-doc/developer/Control-API.md]: Perhaps make anchor links to form a ToC?

Comment (huntc) [inline src/main/play-doc/developer/Control-API.md]: Great idea. I think I'll do that for all pages.

Comment (viktorklang) [inline src/main/play-doc/developer/Control-API.md]: Even better idea! :)

File: src/main/play-doc/developer/Control-API.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+{
+  "requestId": "{request-id}",
+  "bundleId": "{bundleId}"
+}
+```
+
+Field        | Description
+-------------|------------
+requestId    | A UUID for this request.
+bundleId     | An identifier directly related to the digest of a bundle and, if supplied, the digest of its configuration. This identifier is guaranteed to remain related to both the bundle and its optional configuration. It is used for subsequent distinctive references by the API.
```
Comment (viktorklang) [inline src/main/play-doc/developer/Control-API.md:68]: Is it possible to link to somewhere where we explain what the bundleId format looks like?

File: src/main/play-doc/developer/Control-API.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+##### Failure
+
+```
+HTTP/1.1 400 Bad Request
+```
+
+The request was badly formed.
+
+Other status codes should also be treated as a failure.
```
Comment (viktorklang) [inline src/main/play-doc/developer/Control-API.md]: May I suggest editing this to say "Other Http Error codes also apply"?

File: src/main/play-doc/developer/Control-API.md (hunk 2 above)
Comment (huntc) [inline src/main/play-doc/developer/Control-API.md:68]: I think we want to leave the bundle id format explained as being opaque...

File: src/main/play-doc/developer/Control-API.md (hunk 3 above)
Comment (huntc) [inline src/main/play-doc/developer/Control-API.md]: Is this better:

```
Other non 2xx status codes should also be treated as a failure.
```

**Pull request merged**